configuration/configuration.go

```diff
@@ -247,10 +252,13 @@ func (storage Storage) Type() string {
 		case "redirect":
 			// allow configuration of redirect
 		default:
-			return k
+			if storageType != "" {
+				panic("multiple storage drivers specified in configuration")
```

Comment:
Yes.  Perhaps the panic message could make it a bit clearer; either changing the wording ("multiple storage drivers specified in configuration or environment") or printing which drivers have been specified e.g. "multiple storage drivers specified in configuration: filesystem, s3"